cmd/tidb/main.go

```diff
@@ -46,12 +46,12 @@ func main() {
 	switch *clientCase {
 	case "bank":
```

Comment:
So, in m1 we only need to support `bank` and `long_fork`?

Reply:
yeah.